sessionctx/variable/sysvar.go

```diff
@@ -741,6 +749,7 @@ var defaultSysVars = []*SysVar{
 	{Scope: ScopeGlobal | ScopeSession, Name: TiDBAnalyzeVersion, Value: strconv.Itoa(DefTiDBAnalyzeVersion), Type: TypeInt, MinValue: 1, MaxValue: 2},
 	{Scope: ScopeGlobal | ScopeSession, Name: TiDBEnableIndexMergeJoin, Value: BoolToOnOff(DefTiDBEnableIndexMergeJoin), Type: TypeBool},
 	{Scope: ScopeGlobal | ScopeSession, Name: TiDBTrackAggregateMemoryUsage, Value: BoolToOnOff(DefTiDBTrackAggregateMemoryUsage), Type: TypeBool},
+	{Scope: ScopeGlobal | ScopeSession, Name: TiDBMultiStatementMode, Value: Warn, Type: TypeEnum, PossibleValues: []string{BoolOff, BoolOn, Warn}},
```

Comment:
It is a bit weird to have `[]string{BoolOff, BoolOn}...` here, considering we declare that `"BoolOff is the canonical string representation of a boolean false."`

IMHO it's better to define another `Off = "OFF"` and `On = "ON"`?

Reply:
I've made this change. In a future PR I will merge BoolOff and Off, because they can be harmonized. But since I plan to cherry-pick this PR, I'm going to keep it simple and only do that for master.